### DISCUSSION TITLE
Bricked Printer with Display Rev. 1.4

### DISCUSSION THREAD

**Discussion by lexlix** [Jyers/Marlin#2004] [Q&A]
I just got my Ender 3 V2 and am all new to the way the firmware works. I wanted to use JyersUI right from the beginning. Just did 1 test Print with the default firmware to test that the hardware works. I played around a lot and figures out that my display is the new Rev. 1.4. I found the fix to rename the folder private instead of DWIN_SET but after flashing I only got a black screen when booting. I found the fix for the Rev 1.2 Display (https://github.com/Jyers/Marlin/discussions/1768#discussioncomment-2078494) and installed it. It got my display back to booting and showing something but it is very broken. Only CN no EN is working and even the CN lang is very broken.

Could someone assist me with a firmware file for the Rev 1.4 display and the 4.2.2 mainboard to get my printer back in a working state?
I would also be happy with the default Display rev 1.4 firmware just to get ist back working, because flashing the original firmware from Creality does only result in a black screen.

![IMG_2548](https://user-images.githubusercontent.com/6112217/183248844-1fd34aee-c2a1-4003-ab39-ad06384cce1b.jpg)

**Comment by doubleO00** [comment]
https://github.com/Jyers/Marlin/discussions/1768#discussioncomment-3340435
